pkg/config/getters.go

```diff
@@ -0,0 +1,138 @@
+package config
```

Comment:
Yes I think we should aim to remove these, however I held off on doing so in this change because of how deeply they are ingrained in our code base. Would it be alright if we removed these over time in subsequent PRs?